test/interop/python/initFile/cleanupAtExit.prediff

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+sed '/from cleanupAtExit.c:[0-9]*[:]*$/d' $2 > $2.tmp
+sed '/^ *$/d' $2.tmp > $2
+rm $2.tmp
+
+# No need to set PYTHONPATH since we're in the same directory as the generated
+# Python package.
+unset PYTHONPATH
```

Comment:
So we're doing something specifically to set the PYTHONPATH?  Where is that happening, do you remember?  Unsetting the PYTHONPATH during testing seems . . . worrisome.  How certain are we this doesn't break a testing environment?

Reply:
That's a good point. To be safe, I can save and then restore it.

Reply:
I just removed the `unset` entirely.